src/theme/ods-chart-theme.ts

```diff
@@ -553,16 +577,15 @@ export class ODSChartsTheme {
    *
    * The generated tooltips or popover will implement the Orange Design System.
    *
-   * externalizePopover() needs:
-   * - popoverConfig: the configuration of the externalizePopover feature {@link ODSChartsPopoverConfig}
-   * - popoverDefinition: a renderer {@link ODSChartsPopoverDefinition} of the popover/tooltip
-   *
-   *   {@link ODSChartsPopoverManagers} gives preconfigured renderer
+   * @param popoverConfig the configuration of the externalizePopover feature {@link ODSChartsPopoverConfig}
+   * @param popoverDefinition renderer {@link ODSChartsPopoverDefinition} of the popover/tooltip. **{@link ODSChartsPopoverManagers}** is used to
+   * specify preconfigured renderer for Apache ECharts, Boosted 5 or Boosted 4 renderer.
    *
-   *   default value is  {@link ODSChartsPopoverManagers.NONE}: uses default Apache ECharts template to externalize tooltip/popover HTML element, implementing Orange Design system
+   * Default value is  {@link ODSChartsPopoverManagers.NONE}, that means it uses default Apache ECharts template to externalize tooltip/popover HTML element, implementing Orange Design system.
```

Comment:
```suggestion
   * Default value is {@link ODSChartsPopoverManagers}.NONE, that means it uses default Apache ECharts template to externalize tooltip/popover HTML element, implementing Orange Design system.
```
The link seems to be broken, not sure my workaround works in here.